deepchem/models/torch_models/layers.py

```diff
@@ -3097,7 +3097,7 @@ class MolGANConvolutionLayer(nn.Module):
     def __init__(self,
                  units: int,
                  nodes: int,
-                 activation=F.tanh,
+                 activation=torch.tanh,
```

Comment:
Why is it better to use torch.tanh instead of the tanh from functional?

Reply:
the tanh from functional is deprecated and it raises a warning, so i shifted to torch.tanh as the warning suggested